### DISCUSSION TITLE
Google Home Speak Text

### DISCUSSION THREAD

**Discussion by johnlindquist** [johnlindquist/kit#233] [Share]
[Install google-home-speak](https://scriptkit.com/api/new?name=google-home-speak&url=https://gist.githubusercontent.com/johnlindquist/995c76d4d3999568241ab0eb22760c98/raw/f11eeddd2ae73522543ab596fd775992dbe9600a/google-home-speak.js)

```js
// Menu: Google Home Speak Text
// Description: Tell Google Home to speak a message
// Author: John Lindquist
// Twitter: @johnlindquist

let GoogleHome = await npm("google-home-push")
// Find your device IP on your router or
// Home App -> Device -> Settings Gear -> Device Information
let home = new GoogleHome("10.0.0.3")

home.speak(await arg("Speak:"))

```

**Comment by lannonbr** [comment]
Tested this with my Google Nest Hub and it works great! The only thing I wonder is if there is some way with `google-home-push` to mute the bell ding when a message comes into it. I was looking through the source and I am not sure if I can.

**Reply by johnlindquist** [reply]
Please report back if you find anything. I assumed it was impossible, but I'd love to be wrong!
